### PR TITLE
q-dir: Add version 7.99

### PR DESCRIPTION
**Pull request by worldadventurer**
Another freeware utility from softwareok - this one is a quad window file manager

### CODE REVIEW

File: bucket/q-dir.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            ],
+            "shortcuts": [
+                [
+                    "Q-Dir.exe",
+                    "Q-Dir"
+                ]
+            ]
+        }
+    },
+    "pre_install": "mv $original_dir\\Q-Dir\\Q-Dir*.* $original_dir\\",
```
Comment (linsui) [inline bucket/q-dir.json]:
```suggestion
    "extract_dir": "Q-Dir",
```

**Comment by worldadventurer** [comment]
> What's the `Favoriten` directory? Is it needed to be persist?
As far as I can tell (have been using this software for years), the Favoriten directory is a suggested directory for storing Favorites from inside the software. But, it's not even the default directory for when you go to save a Favorite item for the first time. We could delete the Favoriten directory if that's a best practice for scoop?

**Comment by linsui** [comment]
> But, it's not even the default directory for when you go to save a Favorite item for the first time.

Do you mean there is another default directory? 😕  I test and a new item is saved in the `Favoriten` directory. If this is the defauld location, it should be persist.

**Comment by worldadventurer** [comment]
I see your point now. :) Ok, I just added the Favoriten folder to persist. Thanks for catching that!

File: bucket/q-dir.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                [
+                    "Q-Dir.exe",
+                    "Q-Dir"
+                ]
+            ]
+        }
+    },
+    "extract_dir": "Q-Dir",
+    "persist": [
+        "Q-Dir.ini",
```
Comment (Ash258) [inline bucket/q-dir.json]: Move Favoriten folder as first item

File: bucket/q-dir.json [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                [
+                    "Q-Dir_x64.exe",
+                    "Q-Dir"
+                ]
+            ]
+        },
+        "32bit": {
+            "url": "https://www.softwareok.com/Download/Q-Dir_Portable.zip",
+            "hash": "850e2e17dcb88ac808ab7d25789fa5f8f552341d75ecf04af936188f144223a6",
+            "bin": [
```
Comment (Ash258) [inline bucket/q-dir.json]: "bin": "Q-Dir.exe",

File: bucket/q-dir.json (hunk 2 above)
Comment (worldadventurer) [reply bucket/q-dir.json]: ok, done

File: bucket/q-dir.json (hunk 3 above)
Comment (worldadventurer) [reply bucket/q-dir.json]: ok, done

**Comment by worldadventurer** [comment]
Hi @Ash258, would you please re-review after I made the changes you suggested? :)

**Pull request merged**